src/polycap-source.c

```diff
@@ -75,9 +75,9 @@ polycap_photon* polycap_source_get_photon(polycap_source *source, polycap_rng *r
 	r = polycap_rng_uniform(rng);
 	if(r < 0.25)
 		phi = phi;
-	if(r >= 0.25 & r < 0.5)
```

Comment:
Je moet hier `&&` gebruiken ipv `&`. Deze laatste is een [bitwise operator](https://en.wikipedia.org/wiki/Bitwise_operations_in_C) en doet iets anders dan je zou verwachten.

De haakjes mogen weg trouwens.